skimage/future/manual_segmentation.py

```diff
@@ -49,6 +50,9 @@ def manual_polygon_segmentation(image, alpha=0.4, return_all=False):
     labels : array of int, shape ([Q, ]M, N)
         The segmented regions. If mode is `'separate'`, the leading dimension
         of the array corresponds to the number of regions that the user drew.
+    contours : array of float, shape (Q, 2)
```

Comment:
The shape of `contours` is clearly not correct here. Has to be a list of lists or something similar.